apis/securesourcemanager/v1alpha1/types.generated.go

```diff
@@ -37,7 +41,7 @@ type Instance_PrivateConfig struct {
 
 	// Required. Immutable. CA pool resource, resource must in the format of
 	//  `projects/{project}/locations/{location}/caPools/{ca_pool}`.
-	CaPool *string `json:"caPool,omitempty"`
+	CaPoolRef *refs.PrivateCACAPoolRef `json:"caPoolRef,omitempty"`
```

Comment:
Since this contains manual change, it should be moved to _types.go

Reply:
moved to instance_types.go